### PR TITLE
Create builder service

### PR DESCRIPTION
**Pull request by DaemonicWrath**
this is a striped down version of my capstone. just contains the bare minimum that we need to show what it does.

Uses sqs and s3. 
s3 pushes notifications to sqs. 
To build the images Docker in docker is used as this is expected to be run without access to the host docker

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -102,3 +102,6 @@ dist
 
 # TernJS port file
 .tern-port
+
+# Openkata ignore
+lib
```
Comment (ztroop) [inline .gitignore:107]: What is lib? My memory betrays me.

File: dev/miniosqs/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  // send entire app down. Process manager will restart it
+  process.exit(1);
+});
+
+const sqs = new AWS.SQS({
+  endpoint: 'http://alpine-sqs:9324/',
+  accessKeyId: '<accessKeyId>',
+  secretAccessKey: '<secretAccessKey>',
+  region: '<region>',
+  s3ForcePathStyle: true, // needed with minio?
```
Comment (ztroop) [inline dev/miniosqs/src/index.ts:18]: AWS doesn’t allow by default bucket names with dots, I remember reading an article around this topic. True is recommended.

Comment (DaemonicWrath) [reply dev/miniosqs/src/index.ts:18]: ah okay cool this dev/miniosqs is just a small util wrapper so minio and my sqs emulator can talk to one another for some dumb reason minio doesn't support sending events directly to sqs

File: .gitignore (hunk 1 above)
Comment (DaemonicWrath) [reply .gitignore:107]: its the build dir of libraries. we don't want to track build artifacts

**Pull request merged**